<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="search_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>95</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>800</width>
      <height>5</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>5</width>
      <height>650</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>800</width>
      <height>5</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>5</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>5</width>
      <height>650</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>5</width>
      <height>650</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>1000</width>
      <height>5</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>690</width>
      <height>5</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>152</y>
      <width>199</width>
      <height>435</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>本地音乐</string>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>85</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>音乐文件夹</string>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="bar_horizontalSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>600</y>
      <width>421</width>
      <height>40</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLabel" name="pos_time_label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>600</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="char_label">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>600</y>
      <width>10</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="duration_label">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>600</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QListWidget" name="music_listWidget">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>152</y>
      <width>800</width>
      <height>435</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="volume_horizontalSlider">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>95</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="music_name_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>35</y>
      <width>580</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;PMingLiU&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play_pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>600</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next_pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>600</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lrc_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>600</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>词</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="close_pushButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shrink_pushButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Icon_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>35</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="search_pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>95</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="vol_pushButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>95</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="show_musicname_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>605</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="play_style_pushButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>95</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pixmap_pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>600</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>图</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixmap_text_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>1000</width>
      <height>552</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixmap_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;图&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="text_listWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>99</y>
      <width>350</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <zorder>search_lineEdit</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>listWidget</zorder>
   <zorder>bar_horizontalSlider</zorder>
   <zorder>pos_time_label</zorder>
   <zorder>char_label</zorder>
   <zorder>duration_label</zorder>
   <zorder>music_listWidget</zorder>
   <zorder>volume_horizontalSlider</zorder>
   <zorder>music_name_label</zorder>
   <zorder>previous_pushButton</zorder>
   <zorder>play_pushButton</zorder>
   <zorder>next_pushButton</zorder>
   <zorder>lrc_pushButton</zorder>
   <zorder>close_pushButton</zorder>
   <zorder>shrink_pushButton</zorder>
   <zorder>search_pushButton</zorder>
   <zorder>vol_pushButton</zorder>
   <zorder>show_musicname_label</zorder>
   <zorder>play_style_pushButton</zorder>
   <zorder>pixmap_pushButton</zorder>
   <zorder>Icon_label</zorder>
   <zorder>pixmap_text_label</zorder>
   <zorder>pixmap_label</zorder>
   <zorder>text_listWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>bar_horizontalSlider</tabstop>
  <tabstop>music_listWidget</tabstop>
  <tabstop>search_lineEdit</tabstop>
  <tabstop>volume_horizontalSlider</tabstop>
  <tabstop>previous_pushButton</tabstop>
  <tabstop>play_pushButton</tabstop>
  <tabstop>next_pushButton</tabstop>
  <tabstop>lrc_pushButton</tabstop>
  <tabstop>close_pushButton</tabstop>
  <tabstop>shrink_pushButton</tabstop>
  <tabstop>search_pushButton</tabstop>
  <tabstop>vol_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
